tests/Processor/Symfony/PathProcessorTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Protung\OpenApiGenerator\Tests\Processor\Symfony;
+
+use InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+use Protung\OpenApiGenerator\Describer\ExampleDescriber\ExampleDescriber;
+use Protung\OpenApiGenerator\Describer\Form\FormFactory;
+use Protung\OpenApiGenerator\Describer\InputDescriber;
+use Protung\OpenApiGenerator\Describer\ObjectDescriber;
+use Protung\OpenApiGenerator\Describer\OperationDescriber;
+use Protung\OpenApiGenerator\Describer\OutputDescriber;
+use Protung\OpenApiGenerator\Model\ModelRegistry;
+use Protung\OpenApiGenerator\Model\Path\Input\HeaderInput;
+use Protung\OpenApiGenerator\Model\Response;
+use Protung\OpenApiGenerator\Processor\Path\Symfony\PathProcessor;
+use Protung\OpenApiGenerator\Processor\Path\Symfony\SymfonyRoutePath;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Routing\RouteCollection;
+
+final class PathProcessorTest extends TestCase
```

Comment:
Now that we have a unit test for this class, please add tests for the happy path as well

Reply:
@dragosprotung Is it enough to have these asserts in new test?
```php
        $actual = $pathProcessor->process($path);

        self::assertCount(1, $actual);
        self::assertContainsOnly(PathOperation::class, $actual);
```

Reply:
We should assert the path operations